### DISCUSSION TITLE
How to apply theme installed from package.json

### DISCUSSION THREAD

**Discussion by dimaslanjaka** [hexojs/hexo#5191] [Question / Answer]
- How to set theme (`_config.yml` at root hexo project) when installed from package.json ?
- How to configure theme (overriding the theme `_config.yml`) when installed from package.json ?

i've been tried read docs [hexo-theme-landscape](https://github.com/hexojs/hexo-theme-landscape). 

- [to enable](https://github.com/hexojs/hexo-theme-landscape#enable)
- [to configure](https://github.com/hexojs/hexo-theme-landscape#configuration)

I still confused how to set other theme, for example `hexo-theme-claudia` ?
I've tried:
- set `theme: claudia` in `_config.yml`
- create `_config.claudia.yml` to configure
but no effect, the theme still using their default configuration.

**Comment by stevenjoezhang** [comment]
It may have some issues if you're trying to override arrays in the configuration file. But other data types (string, number, bool etc.) should be OK.
